ethcore/src/client/traits.rs

```diff
@@ -233,6 +233,8 @@ pub trait BlockChainClient : Sync + Send + AccountData + BlockChain + CallContra
 			.expect("code will return Some if given BlockId::Latest; qed")
 	}
 
+	fn chain(&self) -> Arc<BlockProvider>;
```

Comment:
That triggered another compiler warning for me:
```
warning: missing documentation for a trait method
   --> ethcore/src/client/traits.rs:236:2
    |
236 |     fn chain(&self) -> Arc<BlockProvider>;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
note: lint level defined here
   --> ethcore/src/lib.rs:17:9
    |
17  | #![warn(missing_docs, unused_extern_crates)]
```

Reply:
Yeah, missed this during review. I added docs in another PR so it'll sort itself out soon enough.